Doc/library/pyexpat.rst

```diff
@@ -217,6 +217,12 @@ XMLParser Objects
    Calling ``SetReparseDeferralEnabled(True)`` allows re-enabling reparse
    deferral.
 
+   Note that :meth:`SetReparseDeferralEnabled` has been backported to prior
```

Comment:
Simplify this to something like this (here & similar for the other docs):

```
   Note that :meth:`SetReparseDeferralEnabled` has been backported to some
   prior releases of CPython as a security fix. Check for availability of
   :meth:`SetReparseDeferralEnabled` using :func:`hasattr` if used in code
   running across a variety of Python versions.
```

Reply:
@gpshead done